charts/registry-scanner/templates/configmap.yaml

```diff
@@ -80,8 +80,9 @@ data:
       exclude:
         {{- toYaml .Values.config.filter.exclude | nindent 8 }}
       {{- end }}
-      maxAgeDays: {{ default 0 .Values.config.filter.maxAgeDays }}
-      maxTagsPerRepository: {{ default 0 .Values.config.filter.maxTagsPerRepository }}
+      maxAgeDays: {{ .Values.config.filter.maxAgeDays }}
+      maxTagsPerRepository: {{ .Values.config.filter.maxTagsPerRepository }}
+      maxRepositories: {{ .Values.config.filter.repositoriesPerRegistry }}
```

Comment:
Please let's try to make consistent the key and the value, otherwise we are going to be confusing both.

Reply:
```suggestion
      maxRepositories: {{ .Values.config.filter.maxRepositories }}
```

Reply:
resolved by changing both key and value to `maxRepositoriesPerRegistry`